src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+
+      <label for="biggie" class="wrapped">
```

Comment:
how do you use this class?
```suggestion
      <label for="biggie">
```